packages/scss/src/components/button/mods.scss

```diff
@@ -253,8 +253,32 @@
 	&:not(:disabled) {
 		@include color.borderGradient(135deg, var(--palettes-brand-400), var(--palettes-AI-500));
 
-		.lucca-icon {
+		&:not(.mod-invert) .lucca-icon {
```

Comment:
We often avoid accumulating `:not`, even if it means redefining .lucca-icon a second time in invertIA. Would that work here?

Reply:
That was my first intention, but it seems a bit overkill as the `button.IA` mixin includes quite a lot.
Maybe I can move the :not selector on index and create 3 separate mixins (AI / invert / invertAI ) ?